core/src/main/java/com/linecorp/armeria/client/ClientFactoryOptions.java

```diff
@@ -241,6 +241,13 @@ private static long clampedDefaultMaxClientConnectionAge() {
     public static final ClientFactoryOption<ConnectionPoolListener> CONNECTION_POOL_LISTENER =
             ClientFactoryOption.define("CONNECTION_POOL_LISTENER", ConnectionPoolListener.noop());
 
+    /**
+     * The client-side maximum allowed duration of a connection drain in microseconds.
+     */
+    public static final ClientFactoryOption<Long> CLIENT_CONNECTION_DRAIN_DURATION_MICROS =
```

Comment:
```suggestion
    public static final ClientFactoryOption<Long> CONNECTION_DRAIN_DURATION_MICROS =
```
because this property is located within the ClientFactoryOptions.